src/sentry/search/snuba/backend.py

```diff
@@ -89,6 +88,10 @@ def _query(self, projects, retention_window_start, group_queryset, environments,
                         groupenvironment__first_release__version=version,
```

Comment:
Would you fix my broken code too? 😬 Just needs the same environment ids passed as part of the `Q`